app/controllers/tasks_controller.rb

```diff
@@ -3,4 +3,19 @@ def index
     @tasks_not_yet = Task.where(complete: false)
     @tasks_done = Task.where(complete: true)
   end
+
+  def new
+    @task = Task.new
+    render 'index'
+  end
```

Comment:
ここ多分`def new` 作らないで、indexの方に `@task = Task.new` 記述したらいけると思います！
```suggestion
  def index
    @task = Task.new
    @tasks_not_yet = Task.where(complete: false)
    @tasks_done = Task.where(complete: true)
  end
```

いけるかの確認も含めてお願いします 🙇